lib/config.js

```diff
@@ -99,6 +99,13 @@ const conf = convict({
       default: '15 minutes'
     }
   },
+  refreshToken: {
+    updateAfter: {
+      doc: 'lastUsedAt only gets updated after this delay',
+      format: 'duration',
+      default: 24 * 60 * 60 * 1000
```

Comment:
See https://github.com/larissagaulia/fxa-oauth-server/blob/6d32c2bd0a2b29a80b91eee6e65f94a641946bb3/lib/config.js#L94 

you can write the `default` as words.